server/lib/service/firestore.dart

```diff
@@ -63,6 +67,29 @@ class FirestoreClient {
     );
   }
 
+  /// Logs [isSuccess] and optional [message] of a performed rule with [ruleId].
+  ///
+  /// The [userId] is the `sub` claim of the user's Google ID token.
+  /// See: https://developers.google.com/identity/protocols/oauth2/openid-connect#an-id-tokens-payload
+  ///
+  /// Throws an [ApiRequestError] if Firestore API returns an error.
+  Future<void> addRun(String userId, String ruleId,
```

Comment:
Suggest to rename the method name to `logRunHistory` to be more clear.

Reply:
Okay!